src/components/UserInfoHeader/index.tsx

```diff
@@ -63,16 +63,16 @@ export const UserInfoHeader: FC<UserInfoHeaderPropType> = ({
               <p className='max-w-prose text-sm sm:text-base'>{description}</p>
             </div>
           )}
-          <Link href={`/accounts/${username}`}>
+          <Link href={`/accounts/${username.trim()}`}>
             <a
-              href={`/accounts/${username}`}
+              href={`/accounts/${username.trim()}`}
               className={[
                 "text-purple hover:text-blue transition-colors",
                 "inline-block order-2 sm:order-3",
                 "mt-0 sm:mt-3",
               ].join(" ")}
             >
-              @{username}
+              @{username.trim()}
```

Comment:
I'm wondering if we need the trimming in places like this? If we ensure that it is not possible anymore to submit a username/sensor name etc. with whitespaces, shouldn't this be unnecessary?

Reply:
This is just to avoid displaying spaces in case there are spaces present in the value. This is just a safety measure. While it should happen, it doesn't hurt. Do you think I should rather not do that?

Reply:
Thanks for the in-person the discussion and the new changes. Happy that the trimming now happens only where we connect to the database 👍